src/Renderer/ThreeExtended/Feature2Mesh.js

```diff
@@ -247,6 +248,46 @@ function geometryToExtrudedPolygon(geometry, properties, options, multiGeomAttri
 
     const indices = [];
     // Build indices
+
+    function addPolygon(polygonGeometry, clockWise, offset = 0, count = 0) {
+        const holesOffsets = polygonGeometry.holes.map(h => h.offset);
+
+        // triangulate the top face
+        let start = vertices.length + polygonGeometry.contour.offset + offset;
+        const end = start + count;
+
+        const triangles = Earcut(geom.attributes.position.array.slice(start * 3,
+            start == end ? undefined : end * 3), holesOffsets, 3);
+
+        if (start == end) {
```

Comment:
Why would this happen?

Reply:
In single polygon case, the count is 0, which is the same as `start == end`. Could be better to change it to `count == 0`, my bad.

Reply:
I'd prefer `count` to have a meaningful value in all case instead :)

So the calling site should be replaced by `addPolygon(geometry, isClockWise, 0, geometry.contour.count);`